staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -187,5 +187,10 @@ func (s *DelegatingAuthorizationOptions) getClient() (kubernetes.Interface, erro
 	clientConfig.QPS = 200
 	clientConfig.Burst = 400
 
-	return kubernetes.NewForConfig(clientConfig)
```

Comment:
please rename this commit, upstream PR is https://github.com/kubernetes/kubernetes/pull/74956

Reply:
Actually, this patch is not used in origin anymore. Please double check. But very probably we can drop it. Please do.

Reply:
It is used by kcm and ks afaik. So we cannot drop it.

Reply:
Done